source/clear-linux/tutorials/greengrass.rst

```diff
@@ -176,51 +175,50 @@ cloud and edge.
    `AWS Greengrass developer guide`_
 
 #. To install and configure AWS Greengrass core on edge platform, follow
-   the instructions at `Start AWS Greengrass on the Core Device`_.
+   the instructions at `Start AWS Greengrass on the Core Device`_. In 
+   step 8(b), download the x86_64 Ubuntu configuration of the AWS Greengrass
+   core software.
 
    .. note::
 
       You will not need to run the ``cgroupfs-mount.sh`` script in step #6
       of Module 1 of the `AWS Greengrass developer guide`_ because this is
       enabled already in |CL|.
 
+#. Be sure to download both the security resources and the AWS Greengrass
+   core software.
+
+   .. note::
+
+      Security certificates are linked to your AWS* account.
+
+
 Creating and Packaging Lambda Functions
 =======================================
 
-#. Complete the tutorial at `Configure AWS Greengrass on AWS IoT`_ .
+#. Complete steps 1-4 of the tutorial at `Create and Package Lambda Function`_ . 
 
    .. note::
 
       This creates the tarball needed to create the AWS Greengrass
       environment on the edge device.
 
-#. Assure to download both the security resources and the AWS Greengrass
-   core software.
-
-   .. note::
-
-      Security certificates are linked to your AWS* account.
 
-#. Replace greengrassHelloWorld.py with Greengrass samples:
+#. In step 5, replace greengrassHelloWorld.py with the classification or object detection 
+   Greengrass sample from `Edge-Analytics`Faas`_:
```

Comment:
@psfoley Could you please remove the tickmark before [`]``Faas``_:  I think this is just a typo. 
This should resolve the build error I get when I test your PR.

Reply:
@mvincerx and @bd-dean thank you for reviewing! I removed the extra tick mark in my latest commit.